api/api.go

```diff
@@ -8,22 +8,23 @@ import (
 	"net/http"
 )
 
-func HTTPServer(ctx context.Context, itemTradeHistory store.TradeStore) error {
+func HTTPServer(ctx context.Context, itemTradeHistory store.TradeStore, alertsDefined store.AlertDefStore) error {
 
 	//Create a mux router instance which can be used assign routes to etc.
 	r := mux.NewRouter()
 
 	//Define the routes
 	r.HandleFunc("/item_traded/{id}", GetTradesByItemHandler(ctx, itemTradeHistory)).Methods("GET")
+	r.HandleFunc("/create_alert/{id}", CreateNewAlertHandler(ctx, alertsDefined)).Methods("POST")
```

Comment:
there are some naming conventions which are used for paths. have a look here: https://restfulapi.net/resource-naming/ 
however its fine not follow these conventions as of now.